iceberg/iceberg-handler/src/test/results/positive/col_stats.q.out

```diff
@@ -459,24 +469,43 @@ Stage-0
     Stage-1
       Reducer 2 vectorized
       File Output Operator [FS_8]
-        Select Operator [SEL_7] (rows=9 width=95)
+        Select Operator [SEL_7] (rows=6 width=192)
           Output:["_col0","_col1","_col2"]
         <-Map 1 [SIMPLE_EDGE] vectorized
           SHUFFLE [RS_6]
-            Select Operator [SEL_5] (rows=9 width=95)
+            Select Operator [SEL_5] (rows=6 width=192)
               Output:["_col0","_col1","_col2"]
-              TableScan [TS_0] (rows=9 width=95)
-                default@tbl_ice,tbl_ice,Tbl:COMPLETE,Col:COMPLETE,Output:["a","b","c"]
+              TableScan [TS_0] (rows=6 width=192)
+                default@tbl_ice_puffin,tbl_ice_puffin,Tbl:COMPLETE,Col:PARTIAL,Output:["a","b","c"]
```

Comment:
The column stats state changed from `complete` to `partial`. Is this expected?

Reply:
yes, that is expected as we compute stats for 2 columns out of 3. BTW, the original q test had a typo and did a validation on a wrong table